apps/recipes-react-components/src/DocComponents/FluentDocsContainer.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { DocsContainer, DocsContextProps } from '@storybook/addon-docs';
+import { FluentStoryContext } from '@fluentui/react-storybook-addon';
+import { webLightTheme, FluentProvider } from '@fluentui/react-components';
+
+type FluentDocsContainerProps = {
+  context: FluentStoryContext & DocsContextProps;
+};
+
+export const FluentDocsContainer: React.FC<FluentDocsContainerProps> = ({ children, context }) => {
+  return (
+    <FluentProvider style={{ backgroundColor: 'transparent' }} theme={webLightTheme}>
```

Comment:
Why is the transparent background necessary here?

Reply:
this looks like copy paste of `apps/public-docsite-v9/src/DocsComponents/FluentDocsContainer.stories.tsx` can you please create a feature request to generalize these sb utils under one package to have DRY codebase ?

Reply:
Also removed the transparent style since it wasn't needed.